pipeline/pcler.py

```diff
@@ -351,7 +355,8 @@ def pcler(args):
                     pw = [None, None]
                 field_filtered = {
                     "spin0": nmt.NmtField(mask, map_filtered[:1], beam=pw[0]),
-                    "spin2": nmt.NmtField(mask, map_filtered[1:], beam=pw[1])
+                    "spin2": nmt.NmtField(mask, map_filtered[1:], beam=pw[1],
```

Comment:
Since we account for the pixwin directly in the TF, I think we should remove these `beam=pw` fields

Reply:
Done in [here](https://github.com/simonsobs/SOOPERCOOL/pull/28/commits/fea9e9e9a6762fe2388c6615d065c5b45d41635b)